promisasu/Portal
# Added FAQ and User Guides to the portal repo

**Taiga User Story**
Tg 595

**Taiga Task(s)**
Tg 640, Tg 641
**What did you change?**
Added FAQ page, App User Guide and Portal User Guide to the code repository along with the relevant images used in these web pages.
Added appropriate routes to load these pages.
**How can we test?**
1. Start the server
2. Check FAQ page on localhost:3000/faq
3. Check App User Guide on localhost:/3000/app-user-guide
4. Check Portal User Guide on localhost:3000/portal-user-guide

All images and content must make sense.
